External/Plugins/HaXeContext/Context.cs

```diff
@@ -1113,11 +1088,39 @@ protected override void InitTopLevelElements()
         /// <returns>Package folders and types</returns>
         public override FileModel ResolvePackage(string name, bool lazyMode)
         {
-            if ((settings.LazyClasspathExploration || lazyMode) && majorVersion >= 9 && name == "flash") 
+            if ((settings.LazyClasspathExploration || lazyMode) && majorVersion >= 9 && name == "flash")
                 name = "flash9";
             return base.ResolvePackage(name, lazyMode);
         }
 
+        /// <summary>
+        /// https://haxe.org/blog/importhx-intro/
+        /// Since Haxe 3.3.0
+        /// </summary>
+        /// <param name="package">Package path</param>
+        /// <returns>Imported classes list (not null)</returns>
+        private MemberList ResolveDefaults(string package)
+        {
+            var result = new MemberList();
+            if (GetCurrentSDKVersion() < "3.3.0") return result;
+            var packagePath = string.IsNullOrEmpty(package) ? string.Empty : package.Replace('.', dirSeparatorChar);
+            while(true)
+            {
+                foreach (var it in classPath)
+                {
+                    if (!it.IsValid || it.Updating) continue;
+                    var path = Path.Combine(it.Path, packagePath, "import.hx");
+                    if (!File.Exists(path)) continue;
+                    var model = ASFileParser.ParseFile(new FileModel(path) {Context = this});
```

Comment:
Why parsing it? shouldn't it be already parsed by the classpath  exploration? If not, maybe an exception for this file could be made.

Reply:
Did you read https://haxe.org/blog/importhx-intro/ ?

Reply:
After parsing, these files will be cached.

Reply:
Or you mean that parsing needs to be moved to BuildClassPath?

Reply:
I don't think it gets cached - and yes it could be handled during classpath exploration and kept up to date like other classes.

Reply:
ok, I will try to do otherwise